### ISSUE TITLE
Show the short description of the repository on "Stared" notification.

### ISSUE THREAD

**Issue by acomagu** [devhubapp/devhub#255]
I would like to see this part in DevHub notification also.

![image](https://user-images.githubusercontent.com/1362750/154008142-1591a5a5-ff9c-4ff3-b66e-2e9b3c5844b9.png)

![image](https://user-images.githubusercontent.com/1362750/154008366-9f60c36e-faba-4211-aa88-e2292c78710a.png)

Thanks!

**Comment by brunolemos** [comment]
yep unfortunately that information is not returned by the same api call, so I would need to either make a separate api call for each repository or start using the graphql api. 

but this app is not actively maintained anymore

**Comment by acomagu** [comment]
OK, thanks!  Maybe I might look into the code.
